diff -ruNb a/Makefile b/Makefile
--- a/Makefile	2012-10-17 16:14:35.000000000 -0400
+++ b/Makefile	2013-06-11 19:57:11.843632903 -0400
@@ -1,6 +1,6 @@
-INSTALL_DIR=/usr/local/bin
-MAN_DIR=/usr/local/man/man1
-ETC_DIR=/etc
+INSTALL_DIR=/usr/bin
+MAN_DIR=/usr/share/man/man1
+ETC_DIR=/etc/colordiff
 VERSION=$(shell egrep '^my .version' colordiff.pl |cut -f 2 -d "'")
 DIST_FILES=COPYING INSTALL Makefile README \
 	colordiff.pl colordiffrc colordiffrc-lightbg cdiff.sh BUGS CHANGES colordiff.1 \
@@ -35,13 +35,23 @@
 	else \
 	  install -d ${DESTDIR}${ETC_DIR}; \
 	fi
+	if [ -f ${DESTDIR}${ETC_DIR}/colordiffrc-lightbg ]; then \
+	  mv -f ${DESTDIR}${ETC_DIR}/colordiffrc-lightbg \
+	    ${DESTDIR}${ETC_DIR}/colordiffrc-lightbg.old; \
+	else \
+	  install -d ${DESTDIR}${ETC_DIR}; \
+	fi
 	cp colordiffrc ${DESTDIR}${ETC_DIR}/colordiffrc
+	cp colordiffrc-lightbg ${DESTDIR}${ETC_DIR}/colordiffrc-lightbg
 	-chown root.root ${DESTDIR}${ETC_DIR}/colordiffrc
+	-chown root.root ${DESTDIR}${ETC_DIR}/colordiffrc-lightbg
 	chmod 644 ${DESTDIR}${ETC_DIR}/colordiffrc
+	chmod 644 ${DESTDIR}${ETC_DIR}/colordiffrc-lightbg
 
 uninstall:
 	rm -f ${DESTDIR}${INSTALL_DIR}/colordiff
 	rm -f ${DESTDIR}${ETC_DIR}/colordiffrc
+	rm -f ${DESTDIR}${ETC_DIR}/colordiffrc-lightbg
 	rm -f ${DESTDIR}${INSTALL_DIR}/cdiff
 	rm -f ${DESTDIR}${MAN_DIR}/colordiff.1
 	rm -f ${DESTDIR}${MAN_DIR}/cdiff.1
